repository/l.json

```diff
@@ -465,6 +465,19 @@
 				}
 			]
 		},
+		{
+			"name": "Lazy Unittester",
+			"author": "Connor Philip",
+			"description": "Creates Python unittest stubs",
```

Comment:
You can remove this line and it will pull the repo description from your GitHub repo. This makes it easier to tweak the description in the future.